utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -87,4 +89,30 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
       paint.style = style
     }
   }
+
+  companion object {
+    fun replaceBulletSpan(
```

Comment:
Please add a brief KDoc explaining what this does & returns (e.g. something like: "Transforms and returns a [SpannableStringBuilder] by replacing all [BulletSpan]s that it contains with [CustomBulletSpan]s.").

Reply:
Done

Reply:
Looks good.